va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -672,6 +688,15 @@ def filter_data(
             json.loads(location_types) if location_types is not None else location_types
         )
         disable_timeframe = False
+        
+        # check if user has geoscoping restrictions. If so, filter down to allowed locations
```

Comment:
I think you could take advantage of values_list here, then you don't need the try/except:

```kwargs["user"].location_restrictions.values_list("name", flat=True)```